### PR TITLE
Display average of score field for all reviews

### LINKED ISSUE #3304: Display average of score for reviews in Review outcomes
**Is your feature request related to an existing functionality? Please describe.**

Display scores and an average score to the review outcomes in submissions tables.

**Additional context**

image

**Priority**

- Low priority

**Affected roles**

- Staff
- Reviewers

### PR DESCRIPTION
**Pull request by gmurtaza00**
Adds scores and an average score to the review outcomes in submissions tables.

image

Open to suggestion on this one, when no review has been submitted. We can either remove Avg. score or add a placeholder for 0 average

image

Closes #3304

### CODE REVIEW

File: hypha/static_src/src/sass/apply/custom/_custom.scss [hunk 1]
```diff
@@ -1 +1,7 @@
 // stylelint-disable no-empty-source
+// stylelint-disable selector-class-pattern
+
+.list__item--average_score {
+    color: $color--mid-dark-grey;
+    font-weight: 400;
```
Comment (frjo) [inline hypha/static_src/src/sass/apply/custom/_custom.scss]: Please use variables for font weight as well.

**Review by frjo — CHANGES_REQUESTED** [review]
Looks good, just one very minor fix before I send this to testing.

**Review by frjo — APPROVED** [review]

**Comment by theskumar** [comment]
Thoughts on moving this here, @frjo @gmurtaza00 

Screenshot 2023-04-04 at 07 45 07@2x

**Comment by gmurtaza00** [comment]
> Thoughts on moving this here, @frjo @gmurtaza00
> 
> Screenshot 2023-04-04 at 07 45 07@2x

image
image

Does this seem alright? @theskumar @frjo

**Comment by theskumar** [comment]
That what's I was thinking, the red color was more of an indicator from my annotation tool. You can keep the font color same as the body font `color` (black) and `font-weight` and rest looks good to me.  @gmurtaza00 

Screenshot 2023-04-04 at 18 54 08@2x

File: hypha/static_src/src/sass/apply/custom/_custom.scss [hunk 2]
```diff
@@ -1 +1,10 @@
 // stylelint-disable no-empty-source
+// stylelint-disable selector-class-pattern
+
+.list__item--average_score {
+    color: $color--dark-grey;
+    font-weight: $weight--normal;
+    font-size: map-get($font-sizes, zeta);
+    display: inline;
+    padding-left: 2px;
+}
```
Comment (frjo) [inline hypha/static_src/src/sass/apply/custom/_custom.scss]: `_custom.scss` should be empty in stock hypha. It is used to add your custom css without worrying about merge conflicts.

File: hypha/static_src/src/sass/apply/components/_list.scss [hunk 3]
```diff
@@ -53,19 +53,6 @@
...
-                max-width: initial;
-                overflow: visible;
-                background-color: $color--mist;
-
-                + #{$root}__item--reviewer-outcome {
-                    width: 100%;
-                    text-align: right;
-                }
-            }
-
```
Comment (frjo) [inline hypha/static_src/src/sass/apply/components/_list.scss:68]: Good, this is some old left over css. Thanks for noticing it.

**Review by frjo — CHANGES_REQUESTED** [review]
Move the css from custom file and we are good I think.

**Review by frjo — APPROVED** [review]

**Comment by gmurtaza00** [comment]
Initially NA score was being display as 99 and average was calculated using this score. Rebasing resolved this issue. Now score is displayed correctly and average is also being calculated correctly.

image

**Review by frjo — APPROVED** [review]

**Comment by frjo** [comment]
Closing this since this view is being replaced.

**Pull request closed without merging**